src/main/resources/config.properties

```diff
@@ -16,4 +16,5 @@ Builder.matlab.modelcoverage.support.warning = To generate a Cobertura model cov
 Builder.matlab.exportstmresults.support.warning = To export Simulink Test Manager results, use MATLAB R2019a or a newer release.
 Builder.matlab.runner.script.target.file.linux.name = run_matlab_command.sh
 Builder.matlab.runner.script.target.file.windows.name = run_matlab_command.bat
+build.workspace.computer.not.found = Unable to get the current computer for this build.
```

Comment:
adding @mw-hrastega  to approve this exception message

Reply:
If there is no value in having the word "current", I would remove it. 

"Unable to get the computer for this build."

It might be possible to use "access" instead of "get".

"Unable to access the computer for this build."

Reply:
Done.